baseline/OSPS-AC.yaml

```
@@ -104,7 +109,9 @@ controls:
           - P2.3
           - E1.2
           - E3.3
-    assessment-requirements:
+      - reference-id: PCIDSS
+        identifiers: 
+          - 2.2.1
```

Here it is missing `assessment-requirements`:
```suggestion
          - 2.2.1
    assessment-requirements:
```